tasks/stop-performance-infrastructure.yml

```diff
@@ -0,0 +1,29 @@
+platform: linux
+image_resource:
+  type: docker-image
+  source:
+    repository: eu.gcr.io/census-gcr/gcloud-kubectl
+params:
+  SERVICE_ACCOUNT_JSON:
+  KUBERNETES_CLUSTER:
+  GCP_PROJECT_NAME:
+inputs:
+  - name: census-rm-kubernetes-dependencies-repo
+    path: bash
+    args:
+      - -exc
+      - |
+        cat >~/gcloud-service-key.json <<EOL
+        $SERVICE_ACCOUNT_JSON
+        EOL
+        # Use gcloud service account to configure kubectl
+        gcloud auth activate-service-account --key-file ~/gcloud-service-key.json
+        gcloud container clusters get-credentials ${KUBERNETES_CLUSTER} --zone europe-west2 --project ${GCP_PROJECT_NAME}
+
+        # Stop Postgresql instance
+        SQL_NAME=`gcloud sql instances list --format="value(name)"`
+        echo $SQL_NAME
+        gcloud sql instances patch $SQL_NAME --activation-policy NEVER
+        # Scale down cluster
+        gcloud container clusters resize $(KUBERNETES_CLUSTER) --num-nodes 0 --node-pool=default-node-pool --region=europe-west2 --quiet
```

Comment:
is there a way we can query gcloud to see what node pools are currently existing? we could just loop through that and resize which makes the code a bit more re-usable